ElectricityService.Core/Models/Ship.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace ElectricityService.Core.Models
+{
+   public class Ship
+    {
+
+        [Key]
+        public string Serial { get; set; }
+        public string Name { get; set; }
+
+        [IgnoreDataMember]
+        public ShipService ShipService { get; set; }
```

Comment:
This should not be on a ship?

Reply:
you said 'look in my project for it' so i did.